examples/list-files-and-folders/Dependencies.toml

```diff
@@ -0,0 +1,309 @@
+# AUTO-GENERATED FILE. DO NOT MODIFY.
```

Comment:
No need to commit the `Dependencies.toml` for examples

Reply:
Removed.